msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -133,6 +133,17 @@ To implement messaging extensions, start with [build a messaging extension](../m
 
 The Teams meeting app extensibility allows you to design your app based on participant roles in a meeting.
 
+### Supported capabilities in meeting
+
+| Capability | Support available | In-tenant user type on mobile | Guest user on mobile | Federated or external user on mobile |
+| Scheduled private meetings | ✔️ | | | |
+| Standard channel meetings | ✔️ | | | |
+
+> [!NOTE]
+> * The overall user experience in standard channel meetings will be same as in scheduled private meetings.
+> * On mobile, anonymous users will not be able to access apps in standard channel meetings.  
+> * Apps on the channel will not be available in the meeting by default.
```

Comment:
We should mention about the App Manifest changes to support apps in channels. There are no specific changes for support in channel meetings though.

Reply:
Mention about GetMeeting and Meeting Start/End Event APIs are currently NOT supported in channel meetings